pkgs/by-name/ci/ciscoPacketTracer7/package.nix

```
@@ -84,7 +84,7 @@ in stdenv.mkDerivation {
     homepage = "https://www.netacad.com/courses/packet-tracer";
     sourceProvenance = with sourceTypes; [ binaryNativeCode ];
     license = licenses.unfree;
-    maintainers = with maintainers; [ lucasew ];
+    maintainers = [];
```

In general, we keep in the `with maintainers;` even if this is empty:
```suggestion
    maintainers = with maintainers; [ ];
```